migrations/20231121203719-DEVSU-2129-datafix-unreviewed-access.js

```diff
@@ -39,14 +45,14 @@ module.exports = {
         );
       }
 
-      unreviewedAcessGroup = unreviewedAcessGroup[0];
+      unreviewedAccessGroup = unreviewedAccessGroup[0];
 
       const userUpdateList = await queryInterface.sequelize.query(
         // eslint-disable-next-line no-multi-str
         'select distinct u.id from users u\
           join user_group_members ugm on (u.id = ugm.user_id)\
           join user_groups ug on (ugm.group_id = ug.id)\
-          where ug.name = \'Bioinformatician\' or ug.name = \'Report Manager\'',
+          where not ug.name = \'Clinician\' and not ug.name = \'Collaborator\'',
```

Comment:
isn't this datafix for a previous ticket? what is this change for?

Reply:
it was supposed to match the change to germline, reverting change @elewis2